src/hooks/useNodeJsContributorsApi.tsx

```diff
@@ -57,107 +67,107 @@ async function getMaxContributors(): Promise<[number, number]> {
  * @param randomPage
  */
 async function getContributor(randomPage: number): Promise<Contributor> {
-  const response = await fetch(`${CONTRIBUTORS_API_URI}&page=${randomPage}`);
-  const contributor = (await response.json())[0] as ContributorApiResponse;
-
-  return {
-    avatarUri: contributor.avatar_url,
-    commitsListUri: `https://github.com/nodejs/node/commits?author=${contributor.login}`,
-    contributionsCount: contributor.contributions,
-    login: contributor.login,
-    profileUri: contributor.html_url,
-  };
+  const response = await fetch(
```

Comment:
We should stick with async/await because it's more readable. 
```suggestion
async function getContributor(randomPage: number): Promise<Contributor> {
  const response = await fetch(
    `${CONTRIBUTORS_API_URI}&page=${randomPage}`
  );
  const jsonResponse = await response.json() as ContributorApiResponse[];

  const contributorData: Contributor[] = jsonResponse.map(
    ({ avatar_url, login, contributions, html_url }) => ({
      avatarUri: avatar_url,
      login,
      contributionsCount: contributions,
      profileUri: html_url,
      commitsListUri: `https://github.com/nodejs/node/commits?author=${login}`,
    })
  );

  const contributor = contributorData.shift() as Contributor;

  if (window.localStorage) {
    window.localStorage.setItem(
      'contributors',
      JSON.stringify(contributorData)
    );
  }

  return contributor;
}

```